ansible/roles/bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+- name: Ensures bootkube dir exists
+  file: path=/tmp/bootkube state=directory
+
+- name: Extract bootkube binaries
+  unarchive:
+    src: https://github.com/kubernetes-incubator/bootkube/releases/download/v0.3.9/bootkube.tar.gz
+    dest: /tmp/bootkube
+    remote_src: True
+
+- name: Render bootkube manifests
+  command: /tmp/bootkube/bin/linux/bootkube render --asset-dir=/tmp/bootkube/assets --experimental-self-hosted-etcd --etcd-servers=http://10.3.0.15:2379 --api-servers=https://{{ inventory_hostname }}:443
```

Comment:
Last question @janwillies - if we want to make use of this prior to the pivot to kubeadm which will support `discovery-file` and `discovery-url` concepts, and leverage what we have here but with multiple API servers, can you help with the following:

1) Can I do this with whats proposed in this PR, switching out `inventory_hostname` for a DNS name that responds with multiple records that correspond to my API server, for instance, a fixed response pointing at 3 IPs belong to 3 hosts I've carved out for being master targets, as long as one of them is the `bootstrap` target where this is running.

2) Can you articulate what that record should look like for two servers (A records, SRV records, or ...)